docs/developer/other/IDE_INTEGRATION.md

```diff
@@ -0,0 +1,31 @@
+# IDE Integration
+
```

Comment:
I set up copies of uPortal often. Even knowing what to set in the setup wizard, it's a bit of a chore. I appreciate this improvement. :1st_place_medal: